content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -23,14 +23,12 @@ Firmament models the scheduling problem as a constraint-based optimization over
 
 Poseidon-Firmament scheduler runs alongside the default Kubernetes scheduler as an alternate scheduler. You can simultaneously run multiple, different schedulers.
 
-## Key Advantages
+Flow graph scheduling with the Poseidon-Firmament scheduler provides the following advantages:
 
-### Flow graph scheduling based Poseidon-Firmament scheduler provides the following key advantages:
-
-- Workloads (pods) are bulk scheduled to enable scheduling at massive scale..
+- Workloads (pods) are bulk scheduled to enable scheduling at massive scale.
 - Based on the extensive performance test results, Poseidon-Firmament scales much better than the Kubernetes default scheduler as the number of nodes increase in a cluster. This is due to the fact that Poseidon-Firmament is able to amortize more and more work across workloads.
-- Poseidon-Firmament Scheduler outperforms the Kubernetes default scheduler by a wide margin when it comes to throughput performance numbers for scenarios where compute resource requirements are somewhat uniform across your workload (Deployments, ReplicaSets, Jobs).
-  Poseidon-Firmament scheduler end-to-end throughput performance numbers, including bind time, consistently get better as the number of nodes in a cluster increase. For example, for a 2,700 node cluster (shown in [Latest Benchmarking Results](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/benchmark/README.md#latest-benchmarking-results), Poseidon-Firmament scheduler achieves a 7× or greater end-to-end throughput than the Kubernetes default scheduler, which includes bind time.
+- Poseidon-Firmament Scheduler outperforms the Kubernetes default scheduler by a wide margin when it comes to throughput performance for scenarios where compute resource requirements are somewhat uniform across your workload (Deployments, ReplicaSets, Jobs).
+  Poseidon-Firmament scheduler end-to-end throughput performance, including bind time, gets better as the number of nodes in a cluster increase. For example, for a 2,700 node cluster (shown in [Latest Benchmarking Results](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/benchmark/README.md#latest-benchmarking-results), the Poseidon-Firmament scheduler achieved a 7× or greater end-to-end throughput than the Kubernetes default scheduler, which includes bind time.
```

Comment:
@sftim , Would be good to fix the last few nits and then push these changes in.
This latest change has a couple of problems:
Line 31: Is this supposed to be an item in the list?
Line 30: The Poseidon-Firmament scheduler outperforms the ... 
Line 31: The link in the second sentence is not set up correctly.
Line 31: @anvithks , Does this make sense?
The Poseidon-Firmament's end-to-end throughput performance and bind time improves as the number of nodes in a cluster increase.

Reply:
:ok_hand: